src/NzbDrone.Core/Datastore/Migration/163_add_on_delete_to_emby_notifications.cs

```diff
@@ -0,0 +1,16 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(163)]
+    public class add_on_delete_to_emby_notifications : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.Sql("UPDATE Notifications SET OnSeriesDelete = 1 WHERE Implementation = 'MediaBrowser'");
+            Execute.Sql("UPDATE Notifications SET OnEpisodeFileDelete = 1 WHERE Implementation = 'MediaBrowser'");
+            Execute.Sql("UPDATE Notifications SET OnEpisodeFileDeleteForUpgrade = 1 WHERE Implementation = 'MediaBrowser'");
```

Comment:
I think this is unnecessary if `On Upgrade` events are being sent, as this would cause multiple events to be sent back to back for the same series.

Though overall I think people can opt into this behaviour instead of forcing it.